tests/auth_tests/test_validators.py

```diff
@@ -259,3 +264,71 @@ def test_ascii_validator(self):
             with self.subTest(invalid=invalid):
                 with self.assertRaises(ValidationError):
                     v(invalid)
+
+    def test_username_minimum_length_validator(self):
+        valid_usernames = ['guido', 'raymond']
+        invalid_usernames = ['1t', 'tw', '']
+        v = validators.UsernameMinimumLengthValidator(min_length=3)
+        for valid in valid_usernames:
+            with self.subTest(valid=valid):
+                v(valid)
+        msg = 'This username is too short. It must contain at least 3 characters.'
+        for invalid in invalid_usernames:
+            with self.subTest(invalid=invalid):
+                with self.assertRaisesMessage(ValidationError, msg):
+                    v(invalid)
+
+    def test_username_validators_help_text(self):
+        validator_config = [
+            {'NAME': 'django.contrib.auth.validators.UnicodeUsernameValidator'},
+            {'NAME': 'django.contrib.auth.validators.UsernameMinimumLengthValidator', 'OPTIONS': {
+                'min_length': 3,
+            }},
+        ]
+        ASCII_and_unicode = _(
+            'Enter a valid username. This value may contain only English letters, '
+            'numbers, and @/./+/-/_ characters.'
+        )
+        length = 'Your username must contain at least 3 characters.'
+        validators_list = get_username_validators(validator_config)
+        help_texts = username_validators_help_texts(validators_list)
+        self.assertEqual(help_texts[0], ASCII_and_unicode)
+        self.assertEqual(help_texts[1], length)
+
+
+class UsernameValidationTests(SimpleTestCase):
+    def test_get_default_username_validators(self):
+        validators_list = get_default_username_validators()
+        self.assertIsInstance(validators_list[0], validators.UnicodeUsernameValidator)
```

Comment:
I think you should do more than just test that the function `get_default_username_validators()` gets the validators. Rather, I think the pipeline all the way down to `AbstractUser` needs to be tested, since you made changes there. (What if somebody refactors `AbstractUser` later?) This would involve creating some users.